{
  try {
    createTable(createStmt);
    fail("Query should be invalid but no error was thrown. Query is: " + createStmt);
  }
 catch (  RuntimeException e) {
    Throwable cause=e.getCause();
    assertTrue("The exception should be a ConfigurationException",cause instanceof ConfigurationException);
    assertEquals(errorMsg,cause.getMessage());
  }
}
